LibertyDSNP/example-client
# timestamp + tests

Purpose
---------------
bug
[https://www.pivotaltracker.com/story/show/179957621](https://www.pivotaltracker.com/story/show/179957621)
As a user, I want to see when posts were published.

Solution
---------------
check out content.ts and find bug in saving the timestamp to the store.
make sure it appears correctly in the feed
is there a better way to test?


Change summary
---------------
* found bug and fixed if statement to return the correct timestamp.
* 


Steps to Verify
----------------
1. create post
2. check that timestamp is correct
3. see posts coming in from other users and on other networks and see that timestamp is correct


Additional details / screenshot
----------------
<img width="1529" alt="Screen Shot 2021-10-18 at 9 24 42 AM" src="https://user-images.githubusercontent.com/43625033/137770794-f2dda2b8-5b2d-45ab-a2f0-da63da8e6273.png">
